R/bootstrap-model.R

```diff
@@ -173,26 +165,27 @@ setup_bootstrap_run <- function(
 
     if(is.null(data)){
       # Only include subjects that entered the original problem by default
-      can_be_joined <- can_be_nm_joined(orig_mod, .join_col = .join_col)
-      if(isTRUE(can_be_joined)){
-        starting_data <- nm_join(orig_mod, .join_col = .join_col) %>%
-          suppressMessages()
+      starting_data <- nm_data(orig_mod, filter = TRUE) %>% suppressMessages()
 
-        # select only columns from original data set
-        starting_data <- starting_data %>%
-          dplyr::select(attr(starting_data, "nm_join_origin")$data)
-
-        if ("DV.DATA" %in% names(starting_data)) {
-          starting_data <- dplyr::rename(starting_data, "DV" = "DV.DATA")
-        }
-      }else{
+      # NULL if IGNORE/ACCEPT expressions cant be turned into dplyr expressions
+      if(is.null(starting_data)){
         rlang::inform(
           paste(
             "Defaulting to input data, which may include data that doesn't enter",
             "the final problem (i.e. ignored subjects)"
           )
         )
         starting_data <- nm_data(orig_mod) %>% suppressMessages()
+      }else{
+        n_recs_dropped <- attributes(starting_data)$n_records_dropped
+        if(n_recs_dropped > 0){
+          rlang::inform(
+            c(
+              glue("Parsed IGNORE/ACCEPT options in {basename(get_model_path(.boot_run))}"),
+              glue("Dropped {n_recs_dropped} records from starting data")
```

Comment:
I don't think this message is needed; but if printing the status to the console, it's probably more informative to say the number of records retained; more likely that the user will be able to connect with that number; we don't usually track the number dropped.

Reply:
I would think knowing the proportion that was dropped (or proportion that remained) would be useful? Like knowing "I lose ~ 15% of the records" type of thing?